TestApp/Sources/TestApp/main.swift

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import CoreFoundation
```

Comment:
Should we consider keeping this as an extraSwiftcFlag? Won't be surprised if this deters a lot familiar-to-Swift users who are new to SwiftWasm.

Reply:
Users should import `CoreFoundation` explicitly because Foundation imports it with `@_implementationOnly` which means users can choose to link it or not. Currently, there is [an import](https://github.com/apple/swift-corelibs-foundation/blob/6787b8a9912a392234062ff3e1be978acc727d29/Sources/Foundation/RunLoop.swift#L11) without the attribute, so no other platform toolchain doesn't require to import CoreFoundation now, but that only one import statement will be removed in the future.

If a user doesn't use any Foundation API that depends on CoreFoundation, he doesn't need to link CoreFoundation. It results in reducing output binary size. If we always link CoreFoundation using extra flags, the output binary size will be more fat. (libCoreFoundation.a is 3.6 MB)

However,  `@_implementationOnly` attribute is not applied in 5.3 branch, so explicit import is required only when using DEVELOPMENT toolchain.

Reply:
Reported separately as https://github.com/swiftwasm/swift/issues/2110, feel free to update it if I got anything wrong.